### DISCUSSION TITLE
在MobaXterm中输入乱码问题

### DISCUSSION THREAD

**Discussion by Damshine** [RipplePiam/MobaXterm-Chinese-Simplified#18] [Q&A]
在MobaXterm中输入的时候乱码怎么解决

**Comment by RipplePiam** [comment]
这个问题过于宽泛了，在提问时请尽可能提供多的相关信息，以便大家考虑这个问题。
其中一种情况的乱码是在bash环境中输入了中文字符命令，无法识别的报错；
还有一些如不可打印字符、当前字符集中不存在的字符等。

与此相关的不属于此汉化项目的问题，直接在网上搜索会有更多的解决方案。
